src/tests/ftest/soak/soak_utils.py

```diff
@@ -584,11 +586,21 @@ def create_ior_cmdline(self, job_spec, pool, ppn, nodesperjob):
         for b_size in bsize_list:
             for t_size in tsize_list:
                 for o_type in oclass_list:
-                    if api in ["HDF5-VOL", "HDF5", "POSIX"] and ppn > 16:
-                        ppn = 16
-                    # DAOS-6095
-                    if api in ["HDF5-VOL", "HDF5"] and t_size == "4k":
-                        t_size = "1m"
+                    # Cancel for ticket DAOS-6095
+                    cancel_api_6095 = ["HDF5-VOL", "HDF5", "POSIX"]
+                    if api in cancel_api_6095 and t_size == "4k" and (
+                            o_type in ["RP_2G1", 'RP_2GX']):
```

Comment:
Optional: PEP8 suggests something like this (https://www.python.org/dev/peps/pep-0008/#indentation):
```suggestion
                    if (api in ["HDF5-VOL", "HDF5", "POSIX"]
                            and t_size == "4k"
                            and o_type in ["RP_2G1", 'RP_2GX']):
```